src/presentational-components/myUserAccess/ResourceDefinitionsLink.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const ResourceDefinitionsLink = ({ onClick, access }) => {
+  return access.resourceDefinitions.length === 0 ? <span>N/A</span> : <Link to={(onClick = { onClick })}>{access.resourceDefinitions.length}</Link>;
```

Comment:
I don't think the link onClick function will work. If you want to pass a function to react-router link it must return a pathname or route object with pathname `{pathname: 'foo'}`.

The link does accept `onClick` handler but it would be better for accessibility to give it `pathname` instead. You can also have both `onClick` to mutate the state and `to` to actually redirect the router.